strategy-builder-gui/src/pages/Settings/components/Stepper/Step.vue

```diff
@@ -0,0 +1,50 @@
+<template>
+  <q-card
+    class="q-pt-md q-pb-lg q-px-lg rounded-borders"
+    :class="status === StepStatus.inProgress ? 'bg-mono-grey-2' : 'bg-mono-grey-1'"
+  >
+    <div class="row justify-between items-center q-mb-md">
+      <div
+        class="text-h6"
+        :class="status === StepStatus.completed ? 'text-mono-green-2' : 'text-mono-grey-3'"
+      >
+        {{ status }}
+      </div>
+      <q-icon
+        :name="`img:${require(`../../assets/status-${
+          status === StepStatus.completed ? 'completed' : 'in-progress'
+        }.svg`)}`"
+        size="20px"
+      />
+    </div>
+    <div class="text-white text-h5 q-mb-xs"> {{ title }} </div>
+    <div class="text-mono-grey-3 text-body1"> {{ desc }} </div>
+  </q-card>
+</template>
+
+<script lang="ts">
+import { defineComponent, PropType } from 'vue';
+
+export enum StepStatus {
```

Comment:
```
  Completed = 'Completed',
  InProgress = 'In Progress',
  NotStarted = 'Not started',
  ```

Reply:
Completed = 'completed',
InProgress = 'in-progress',
NotStarted = 'not-started',